### PR TITLE
Added image to Landing Page

### PR DESCRIPTION
**Pull request by ihardzeenka**
Full Screen
image

Half Screen
image

Mobile Screen
image

### CODE REVIEW

File: data/landing.yaml [hunk 1]
```diff
@@ -20,12 +20,12 @@ hero:
...
-#    filename: "logo_image.png" # filename of your hero image (including file extension)
-#    alt: "Logo" # Optional but recommended
-#    boxShadow: true # default 'false' (excludes .svg images)
-#    rounded: true # round the image corners? default 'false' (excludes .svg images)
+  image:
+    path: "images" # path to image under configured assets directory. default 'images'
+    filename: "abstract-image.png" # filename of your hero image (including file extension)
+    alt: "Logo" # Optional but recommended
+    boxShadow: false # default 'false' (excludes .svg images)
+    rounded: true # round the image corners? default 'false' (excludes .svg images)
```
Comment (DenisKlimenko) [inline data/landing.yaml:28]: Just wondering, is there any reason not to keep the default value here?

Comment (ihardzeenka) [reply data/landing.yaml:28]: It adds image into a square box with shadows
image

Comment (DenisKlimenko) [reply data/landing.yaml:28]: The shadow is added by `boxShadow: false`. The `rounded: true` or `rounded: false` doesn't change anything in visualization cause the corners are transparent. Tested it locally.

**Review by DenisKlimenko — APPROVED** [review]

File: themes/lotusdocs/layouts/partials/landing/hero.html [hunk 2]
```diff
@@ -101,7 +101,7 @@ <h4 class="heading mt-0 mb-3">
                                         {{ .Content | safeHTML }}
                                     {{ else }}
                                         {{ with .Resize (printf "%dx%d webp q85 drawing Lanczos" .Width .Height) }}
-                                            <img src="{{ .RelPermalink }}" class="{{ if $boxShadow }}shadow-lg{{ else }}{{ end }} mx-auto d-block {{ if $rounded }}rounded-top-4{{ else }}{{ end }}" width="{{ .Width }}" height="{{ .Height }}" alt="{{ $alt }}">
+                                            <img src="{{ .RelPermalink }}" class="{{ if $boxShadow }}shadow-lg{{ else }}{{ end }} mx-auto d-block {{ if $rounded }}rounded-top-4{{ else }}{{ end }}" width=100% height=100% alt="{{ $alt }}">
```
Comment (gekola) [inline themes/lotusdocs/layouts/partials/landing/hero.html:104]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#width. % may not be expected here by some browsers. Use embedded css (`style="width: 100%; height: 100%;"`) instead?

Comment (ihardzeenka) [reply themes/lotusdocs/layouts/partials/landing/hero.html]: Fixed

**Review by gekola — APPROVED** [review]

**Review by RGBD — APPROVED** [review]

**Pull request merged**